src/lib/esri-map/README.md

```diff
@@ -1,21 +1,20 @@
 # esri-map
 
-`esri-map` is a component that will create an ESRI map using the [ArcGIS API for JavaScript v4.3](https://developers.arcgis.com/javascript/)
+`esri-map` is a component that will create an ESRI map using the [ArcGIS API for JavaScript v4.6](https://developers.arcgis.com/javascript/)
 
 ## Usage
 
 ```
 import { Component } from '@angular/core';
 
-import { EsriLoaderService } from 'angular-esri-loader';
+import { EsriModuleProvider } from 'angular-esri-components/src/lib/core';
```

Comment:
The from portion of the import is to verbose. It should just be `angular-esri-components/src/lib/core`.